automation_tools/satellite6/upgrade/tests/existence.py

```diff
@@ -0,0 +1,185 @@
+"""Helper functions and variables to test entity existence and associations
+post upgrade
+"""
+
+import os
+import pickle
+from automation_tools.satellite6.upgrade.tasks import csv_reader
+
+# Components for which the post upgrade existence will be validated,
+# non_orged - The components where org is not required to get the data about
```

Comment:
non_orged and orged is not too intuitive to me as org_required and org_not_required, reading the latter I can figured out the idea without reading the docstring.